holmes/plugins/toolsets/bash/kubectl/kubectl_events.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import re
+from typing import Any
+
+from holmes.plugins.toolsets.bash.common.stringify import escape_shell_args
+from holmes.plugins.toolsets.bash.common.validators import regex_validator
+
+
+def create_kubectl_events_parser(kubectl_parser: Any):
+    parser = kubectl_parser.add_parser(
+        "events",
+        help="List events",
+        exit_on_error=False,  # Important for library use
+    )
+    parser.add_argument(
+        "-n",
+        "--namespace",
+        type=regex_validator("namespace", re.compile(r"^[a-z0-9][a-z0-9\-]*$")),
+    )
+    parser.add_argument("-A", "--all-namespaces", action="store_true")
+    parser.add_argument(
+        "-l",
+        "--selector",
+        type=regex_validator("selector", re.compile(r"^[a-zA-Z0-9\-_.=,!()]+$")),
+    )
+    parser.add_argument(
+        "--field-selector",
+        type=regex_validator("field selector", re.compile(r"^[a-zA-Z0-9\-_.=,!()]+$")),
+    )
+    parser.add_argument(
+        "--for", dest="for_object", type=lambda x: _validate_for_object(x)
+    )
+    parser.add_argument("--types", type=lambda x: _validate_event_types(x))
+    parser.add_argument("-w", "--watch", action="store_true")
+    parser.add_argument("--no-headers", action="store_true")
+
+
+def _validate_for_object(value: str) -> str:
+    """Validate the --for object parameter."""
+    MAX_FOR_OBJ_SIZE = 253
+    for_object_pattern = re.compile(r"^[a-zA-Z0-9][a-zA-Z0-9\-_./:]*$")
+
+    if not for_object_pattern.match(value):
+        raise argparse.ArgumentTypeError(f"Invalid for_object: {value}")
+    if len(value) > MAX_FOR_OBJ_SIZE:
```

Comment:
Not sure if truncate the event when max size reach is a better idea instead of raising errors, otherwise we cannot get events on a resource exploded with events.